src/plugins/janus_sip.c

```diff
@@ -2003,6 +2004,27 @@ int janus_sip_init(janus_callbacks *callback, const char *config_path) {
 	}
 	JANUS_LOG(LOG_VERB, "Local IP set to %s\n", local_ip);
 
+	/*
+		Since we might have to derive SDP connection address from local_media_ip make sure it has a meaningful value
+		for the prupose of using it in the SDP c= header.
+	*/
```

Comment:
Please don't use this format for comments. Align to what we use in other parts of the code, so something the lines of

    /* Since ...
     * for the ... */

As a side notem there's a typo in "purpose".

Reply:
The point taken.

Reply:
The format of the comment is still incorrect.

Reply:
Sorry about that. Fixed.